Framework/NexusGeometry/inc/MantidNexusGeometry/NexusGeometrySave.h

```diff
@@ -0,0 +1,36 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#ifndef MANTID_NEXUSGEOMETRY_NEXUSGEOMETRYSAVE_H_
+#define MANTID_NEXUSGEOMETRY_NEXUSGEOMETRYSAVE_H_
+
+#include "MantidNexusGeometry/DllConfig.h"
+
+#include <iostream>
```

Comment:
I don't think you require either of these includes

Reply:
MANTID_NEXUSGEOMETRY_DLL depends on include "MantidNexusGeometry/DllConfig.h", and is failing without it